### ISSUE TITLE
[seek help] make error in Debian9

### ISSUE THREAD

**Issue by jianguda** [stevenlovegrove/Pangolin#553]
Hey
Just like #486 , but I have already installed `libegl1-mesa-dev`, `libxkbcommon-dev`, `libwayland-dev` and `wayland-protocols`. My output looks like this:
```
[ 66%] Linking CXX shared library libpangolin.so
/usr/bin/ld: cannot find -lEGL
collect2: error: ld returned 1 exit status
src/CMakeFiles/pangolin.dir/build.make:2680: recipe for target 'src/libpangolin.so' failed
make[2]: *** [src/libpangolin.so] Error 1
CMakeFiles/Makefile2:176: recipe for target 'src/CMakeFiles/pangolin.dir/all' failed
make[1]: *** [src/CMakeFiles/pangolin.dir/all] Error 2
Makefile:149: recipe for target 'all' failed
make: *** [all] Error 2
```
I even do not have `usr/lib/x86_64-linux-gnu/libEGL.so` or `/usr/lib/x86_64-linux-gnu/libEGL.so.1`.
Really confusing :<

**Comment by jianguda** [comment]
Okay, I solved it! For note:
In my case, after I installed libEGL by `apt install libegl1-mesa-dev`, the invalid softlink `libEGL.so` is placed at `/usr/lib/x86_64-linux-gnu/` which actually points to `libEGL.so.1.0.0` at same folder, but only its successor `libEGL.so.1.1.0`. The solution is to remove current `libEGL.so` and recreate new `libEGL.so` pointing to `libEGL.so.1.1.0`.

Takeaway
```
sudo rm /usr/lib/x86_64-linux-gnu/libEGL.so
sudo ln -s /usr/lib/x86_64-linux-gnu/libEGL.so.1.1.0 /usr/lib/x86_64-linux-gnu/libEGL.so
```

We could aslo use following commands to help check:
`stat libEGL.so`
`ld -lEGL --verbose`
